### PR TITLE
Add dot regular expression in unique_id

### PR DESCRIPTION
**Pull request by cp-shruti-s**

### CODE REVIEW

File: jobs/jobs.go [hunk 1]
```diff
@@ -90,6 +91,15 @@ func (repository *CareerRepository) GetCareers() (careersList []Career, err erro
 											   apply_seo_title, apply_seo_description
 											   FROM jobs WHERE is_active = 1 `)
 
+	var uniqueId string
```
Comment (cp-sumi-k) [inline jobs/jobs.go:94]: We should create new API for this script, otherwise it will execute in all API calls

File: vue-frontend/server.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ async function createServer(
       root,
       logLevel: isTest ? "error" : "info",
       server: {
-        middlewareMode: "ssr",
+        middlewareMode: "html",
```
Comment (cp-sumi-k) [inline vue-frontend/server.js:38]: Keep it as SSR mode

File: vue-frontend/src/config.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ export default Object.freeze({
   LINKEDIN_URL: "https://www.linkedin.com/company/canopasinc",
   MEDIUM_URL: "https://medium.com/canopas",
   BLOG_URL: "https://blog.canopas.com",
-  API_BASE: "https://dev-stack-api.canopas.com",
+  API_BASE: "http://localhost:8080",
```
Comment (cp-sumi-k) [inline vue-frontend/src/config.js:9]: Why do we need to change it?

File: vue-frontend/src/config.prod.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ export default Object.freeze({
   LINKEDIN_URL: "https://www.linkedin.com/company/canopasinc",
   MEDIUM_URL: "https://medium.com/canopas",
   BLOG_URL: "https://blog.canopas.com",
-  API_BASE: "https://prod-stack-api.canopas.com",
+  API_BASE: "http://localhost:8080",
```
Comment (cp-sumi-k) [inline vue-frontend/src/config.prod.js:9]: Also for here?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: jobs/jobs.go (hunk 1 above)
Comment (cp-shruti-s) [reply jobs/jobs.go:94]: No need to create separate API for that because if condition in true that time only call update query

Comment (cp-sumi-k) [reply jobs/jobs.go:94]: We have to create separate API for this type of change, in future if we want to disable that API , we can do that

Comment (cp-shruti-s) [reply jobs/jobs.go:94]: okay

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**